tests/scripts/timeout-handling-test.sh

```diff
@@ -98,10 +98,10 @@ fi
 # Check the output from clusterclient
 cmp "$testname.out" <<'EOF' || exit 99
 OK
-unknown error
-unknown error
-unknown error
-unknown error
+error: Timeout
+error: Timeout
```

Comment:
Ideally the test should pass with all supported hiredis versions.

I think at least a comment like `# If used with hiredis < 1.1.0 we get "unknown error" instead of "error: Timeout" here.` could be useful...

Reply:
Good idea!